package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "ilmomasiina",
-  "version": "2.0.0-alpha1",
+  "version": "2.0.0-alpha11",
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": "^14.19.0",
-    "npm": "^6.14.13"
+    "node": "^16",
+    "npm": "^7"
   },
   "scripts": {
     "bootstrap": "pnpm install",
     "clean": "pnpm run -r clean",
-    "build": "pnpm run -r build",
+    "build": "pnpm run -r --workspace-concurrency=1 build",
```

Comment:
Why is this required? I assume there is a good reason, but would be a good idea to document and write it down.

Reply:
Basically, I got lots of heisenbugs (with errors appearing out of nowhere) when multiple `tsc` instances were building the same folder simultaneously. This seems to eliminate that.

Reply:
Unfortunately, `package.json` doesn't have many options for documentation...